library/src/scala/internal/tasty/CompilerInterface.scala

```diff
@@ -853,6 +846,9 @@ trait CompilerInterface extends scala.tasty.reflect.Types {
   // SYMBOLS //
   /////////////
 
+  /** Returns the symbol of the current enclosing definition */
```

Comment:
```suggestion
  /** Returns the symbol of the enclosing definition of the given context */
```

Reply:
I'd suggest avoid `current` in the doc, as it's expected to give meaning to `current`.